OrbitBase/include/OrbitBase/ThreadPool.h

```diff
@@ -60,10 +63,13 @@ class ThreadPool {
   // the thread pool creates a new worker thread if current number of worker
   // threads is less than maximum pool size.
   //
-  // TODO: If queue is empty thread_pool reduces number of worker threads
-  // until they hit thread_pool_min_size
-  static std::unique_ptr<ThreadPool> Create(size_t thread_pool_min_size,
-                                            size_t thread_pool_max_size);
+  // If queue is empty thread_pool reduces number of worker threads
+  // until they hit thread_pool_min_size. thread_ttl specifies the duration
+  // for the such a thread to remain in idle state before it finishes the
+  // execution.
+  static std::unique_ptr<ThreadPool> Create(
+      size_t thread_pool_min_size, size_t thread_pool_max_size,
+      absl::Duration thread_ttl = absl::Milliseconds(50));
```

Comment:
What's the rationale behind the 50ms value, at first glance it seems a bit short to me, shouldn't it be in the order of magnitude of time between user interactions?

Reply:
The rationale is that creating additional threads is not a norm and once load goes down we should probably free them relatively quickly. But maybe I should remove the default here. Because context here is everything and I agree that for user interaction it should probably larger.

Reply:
removed the default.